### PR TITLE
Cascade-fix: update for accessibility #188

### LINKED ISSUE #188: 06-cascade-fix: update exercise to not rely on color
Complete the following REQUIRED checkboxes:
-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this issue follows the `location for request: brief description of request` format, e.g. `Foundations exercises: Add exercise for XYZ`

The following checkbox is OPTIONAL:

-   [ ] I would like to be assigned this issue to work on it

<hr>

**1. Description of the Feature Request:**

The 06 exercise in the foundations directory needs to be updated to not rely on color for the two groups of texts (currently green and red). One suggestion on Discord was to use black and gray or rely on font weight (200 and 900). Another option could be to have all text be black, but have one group have a background color.

The goal should be to use CSS properties mentioned in the [CSS Foundations lesson](https://www.theodinproject.com/lessons/foundations-css-foundations)

**2. Acceptance Criteria:**

- [ ] The exercise is updated to not rely on text color alone to differentiate the two groups of texts, or use a better combination of colors
- [ ] The solution image in the README is updated to reflect the new solution criteria

**3. Additional Information:**

### PR DESCRIPTION
**Pull request by Amandasilvbr**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `01-flex-center: Update self check`

Complete the following checkbox ONLY IF it is applicable to your PR. You can complete it later if it is not currently applicable:
-   [x] I have ensured that the TOP solution files match the Desired Outcome image

<hr>

**1. Because:**

This pull request is for issue 188, which was necessary to make the exercise in question more visually accessible.

Closes #188

**2. This PR:**

* Had its font changed, so that the differentiation between the letter with weight 900 and 200 are more visible
* It had its colors changed by the font-weight property, so that the exercise is not only dependent on the color, which can affect the accessibility performance

**3. Additional Information:**

I've added a comment at the end of README.md explaining to old and new students why the change was made. In addition to clarifying this issue, the notice also seeks to encourage people to pay attention and be interested in this issue, which is not always taken into account in web development.

Also, I would like to make an addendum and leave a recommendation. Although one of the buttons has color, it is not the focus of the exercise, and was designed not to affect the vision of colorblind or low vision people. As you can see below:

image

For comparison, here is an example of misuse of color, which is not accessible, because it has low contrast.:

image

To whom it may concern, the name of this extension for accessibility is Accessibility Insights for Web.

I'm a Brazilian student, so if there are any spelling errors, please contact me.

### CODE REVIEW

**Review by dm-murphy — COMMENTED** [review]
Great work @Amandasilvbr and thanks for taking this issue on so quickly! 👍 

I liked your approach of using the `font-weight` instead of color. I also appreciated you retaining the button with green background and white lettering as an example of an accessible color option that passes WCAG standards. 

I left some comments in the review for you to look over when you get the chance. Please let me know if you have any questions. 

Also pinging @thatblindgeye to see if there are any other thoughts on this one.

File: foundations/06-cascade-fix/README.md [hunk 1]
```diff
@@ -13,3 +13,5 @@ Issues with the cascade can be the bane of their existence for many when it come
 ### Self Check
 - Did you make sure to not edit the HTML file?
 - If you added selectors to the CSS, do they target a valid HTML element?
+
+_Note: this exercise has had its style changed, for accessibility reasons. For new and old students, we propose to carry out the exercise noting how important it is to pay attention to the accessibility of the web, and we encourage more and more practices like this._
```
[the remark this answers is not in the document]
Comment (Amandasilvbr) [reply foundations/06-cascade-fix/README.md:17]: Yes, it´s sound better!

File: foundations/06-cascade-fix/solution/solution.css [hunk 2]
```diff
@@ -1,6 +1,11 @@
+body{
+  font-family:Arial, Helvetica, sans-serif
+}
+
 .para,
 .small-para {
-  color: hsl(0, 100%, 50%);
+  color: hsl(0, 0%, 0%);
+  font-weight: 900;
```
[the remark this answers is not in the document]
Comment (Amandasilvbr) [reply foundations/06-cascade-fix/solution/solution.css:8]: Thank you for bring me this information. I made some tests, and i will alternate te readme img with the weight 700 and 100.

[the remark this answers is not in the document]
Comment (Amandasilvbr) [reply foundations/06-cascade-fix/solution/solution.css:8]: @dm-murphy , can you give me your opinion of the result, please? Do you think it's better the way it was before? I'm undecided, I don't know if it's still clear that the smaller paragraphs are bold.
image

[the remark this answers is not in the document]
Comment (dm-murphy) [reply foundations/06-cascade-fix/solution/solution.css:8]: Thanks @Amandasilvbr! I can distinguish the difference between the fonts here myself but agree it's less pronounced than before. Here's another idea, what about using an `800` value and labeling the text as "extra bold text". Would that be dark enough to make the smaller paragraph stick out better?

[the remark this answers is not in the document]
Comment (Amandasilvbr) [reply foundations/06-cascade-fix/solution/solution.css:8]: Yes, @dm-murphy ! I made a test, and 800 is really better.

[the remark this answers is not in the document]
Comment (Amandasilvbr) [reply foundations/06-cascade-fix/solution/solution.css]: I just made the last changes, if I'm missing something, let me know!

File: foundations/06-cascade-fix/solution/solution.html [hunk 3]
```diff
@@ -9,15 +9,15 @@
...
 
     <button id="confirm-button" class="button confirm">Confirm</button>
     <button id="cancel-button" class="button cancel">Cancel</button>
 
-    <div class="text">I'm a div with green text!</div>
-    <div class="text">I'm a div with green text and a child div!
-      <div class="text child">I'm a smaller child div with red text.</div>
+    <div class="text">I'm a div with thin text!</div>
+    <div class="text">I'm a div with thin text and a child div!
+      <div class="text child">I'm a smaller child div with bold text.</div>
```
Comment (dm-murphy) [inline foundations/06-cascade-fix/solution/solution.html:20]:
```suggestion
      <div class="text child">I'm a smaller child div with extra bold text.</div>
```
**suggestion:** Adding the "extra" to the wording in the `solution.html`. You can commit these right from the comments if you'd like!

File: foundations/06-cascade-fix/solution/solution.html [hunk 4]
```diff
@@ -9,15 +9,15 @@
     <link rel="stylesheet" href="solution.css">
   </head>
   <body>
-    <p class="para">I'm just a paragraph with red text!</p>
-    <p class="para small-para">I'm a smaller paragraph, also with red text!</p>
+    <p class="para">I'm just a paragraph with bold text!</p>
+    <p class="para small-para">I'm a smaller paragraph, also with bold text!</p>
```
Comment (dm-murphy) [inline foundations/06-cascade-fix/solution/solution.html:13]:
```suggestion
    <p class="para small-para">I'm a smaller paragraph, also with extra bold text!</p>
```
**suggestion:** Adding the "extra" to the wording in the `solution.html`. You can commit these right from the comments if you'd like!

**Review by dm-murphy — CHANGES_REQUESTED** [review]
Thanks for the updates @Amandasilvbr! I believe I found a few missing "extra" words in the `solution.html` file so I added those as suggested commits which you can commit right from the comments. Other than that I think everything looks good! Thanks for all the iterations on this one! Your work is much appreciated 🙂

**Review by dm-murphy — APPROVED** [review]
Thanks @Amandasilvbr for the great work on this issue and all your help contributing to The Odin Project! 🎉

**Pull request merged**